vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -43,18 +46,17 @@ class TextShadowNode final : public ShadowNodeBase {
   }
 
   void AddView(ShadowNode &child, int64_t index) override {
+    auto &childNode = static_cast<ShadowNodeBase &>(child);
+    VirtualTextShadowNode::ApplyTextTransformToNode(childNode, textTransform, /* hasChanged = */false);
+
+    auto run = childNode.GetView().try_as<winrt::Run>();
     if (index == 0) {
-      auto run = static_cast<ShadowNodeBase &>(child).GetView().try_as<winrt::Run>();
       if (run != nullptr) {
         m_firstChildNode = &child;
         auto textBlock = this->GetView().as<xaml::Controls::TextBlock>();
-        std::wstring text(run.Text().c_str());
-        transformableText.originalText = text;
-        text = transformableText.TransformText();
-        textBlock.Text(winrt::hstring(text));
-
+        textBlock.Text(run.Text());
```

Comment:
it would be great to be able to use CharacterCasing instead but that's only available for TextBox, not TextBlock :(
Consider filing a xaml bug :)
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textbox.charactercasing?view=winrt-19041

Reply:
Will do :)

Reply:
Actually CharacterCasing is a little different. CharacterCasing actually changes the stored value of the typed text, where we only want to change the rendered value. I'd hate to file a bug for something we don't currently need and doesn't impact our app perf / memory usage significantly :)